### ISSUE TITLE
Timeout on Request no matter what I do

### ISSUE THREAD

**Issue by Andy2000211** [Gkiokan/ps4-remote-pkg-sender#74]
I connected internet to my PC using usb to gigabit ethernet adapter, all good network works.
Connected PS4 to PC with ethernet cable in the PC motherboard's ethernet port and enabled internet sharing from adapter, internet works on PS4 like that.
Config in the PKG Sender:
Server IP: Ethernet Adapter option
Port: 8337(I think it was by default) and tried changing to 2121 because someone said to change port somewhere
Server App: express
PS4 IP all good, says it can connect.
Playstation APP PS4 RPI(FlatZ) downloaded from HB Store
Request timeout changed from 2500ms to 4000ms still gets me the "Timeout on Request" when I want to start sending PKG files.
On PS4 when I check connection status everything is good, only NAT Type Failed. Can that be the problem? Also I use a DNS address to block Sony servers.
Please help me, sorry for posting here if I shouldn't but I am frustrated because nothing seems to work.
![Checking server and PS4](https://github.com/Gkiokan/ps4-remote-pkg-sender/assets/64160189/077dcd68-c140-4d30-89bc-44d03243c70e)
![When I start to send](https://github.com/Gkiokan/ps4-remote-pkg-sender/assets/64160189/a944ecc5-c041-4c87-b18c-c65bffd42827)

**Comment by Gkiokan** [comment]
You didn't mention that you have run flatZ RPI on your PS4.  
It must stay in the foreground. If that doesn't work, try the alternative RPI from nzydark (Misc > Downloads) 
it is rebuild with the OOP SDK and some has reported it works better then the legacy.  

Could you also open the PS API Window? Right top 2nd Icon, and share what the debug logs says.  

Sam Daniels made a Guide Video on a older Version, with connection settings and stuff, you might have a look into that.  
I never touched the Network Settings between PC and PS4 but it is a network issue for sure.  
Just double check that you have allowed the Application to use Network Connection, too. Windows is a b**** on that. 

https://youtu.be/RyKUm5rndPQ?t=135

**Comment by Andy2000211** [comment]
Thank you for answering. The flatZ RPI was running on the PS4. I basically got it working by just connecting both the PC and PS4 directly to the router with a LAN cable and it was pretty fast and using the default port. I was helping a friend so the PS4 isn't at my place anymore otherwise I would've sent a screenshot with "PS API Window" and would've tried the other RPI.
I think the video you linked it's the same as the one I watched and he said I can just connect the PS4 and PC straight to the router and that's what I did and it worked.
Thanks again for answering! Have a good day/night!
